sources/Adapters/picoTracker/system/input.cpp

```diff
@@ -1,3 +1,14 @@
 #include "input.h"
+#include <stdio.h>
 
-uint16_t scanKeys() { return (~gpio_get_all() & 0x0001FF00) >> 8; }
+uint16_t scanKeys() {
+#ifdef USB_REMOTE_UI
```

Comment:
Can you explain what's going on here? I don't fully get it

Reply:
Yeah sorry, will add better comment here 
This is the input side of remote UI, letting client app send key input events over USB serial. But the currently this doesn't work correctly as I'm just sending individual key "events". Instead what I need to do is change the client to send the same bitmask as the current code gets from reading the group of gpio pins and have the code here just treat the incoming byte from serial input as such.